README.adoc

```diff
@@ -6,18 +6,17 @@ A client controlled Wayland compositor.
 Configuration
 -------------
 
-Configuration languages are confusing. They are neither flexible nor easy to use. Moreover, it adds a lot of complexity to the program itself.
-kiwmi listens to a socket and offers a basic client, kiwmic, to communicate with kiwmi. Much like how bspwm works.
+Configuration languages are confusing. They are neither flexible nor easy to use. Moreover, they add significant complexity to the program itself.
+Kiwmi listens to a socket and offers a basic client, kiwmic, to communicate with kiwmi, similarly to how bspwm works.
```

Comment:
As kiwmi is the program and executable name it was lowercased on purpose. Please leave it that way.